addons/packages/ako-operator/1.5.0/README.md

```diff
@@ -56,6 +130,73 @@ The following configuration values can be set to customize the ako-operator inst
 | `avi_advanced_l4` | Optional | describes the settings for the services API usage for L4 |
 | `avi_auto_fqdn` | Optional |  describes the FQDN generation for L4: default,flat,disabled |
 
+### Verify your configurations
+After the package is installed and reconcile successfully, two AKODeploymentConfigs are deployed globally. 
+You can list and inspect them with `kubectl get akodeploymentconfig`. They are the configuration interfaces for the ako-operator.
+In general,
+
+* `install-ako-for-management-cluster` specifies the parameters to deploy AKO statefulset in the management cluster, which
+  then provides L4/L7 load balancing for applications.
+* `install-ako-for-all` specifies the parameters to deploy AKO statefulsets and reconcile users in each future workload clusters. The ako-operator
+  passes these parameters to each workload clusters via addon secrets.
+
+In the management cluster, ensure that there is no error level message in the ako-operator manage log. 
+
+```shell
+# retrieve the pod name of the ako-operator manager
+kubectl get pods -n tkg-system-networking
+
+# check if the log is error-free.
+kubectl logs ako-operator-controller-manager-xxxxxxxx-xxxxx manager -n tkg-system-networking
+```
+
 ## Usage Example
 
-This walkthrough guides you through using ako-operator...
+### Create an L4 Service in management cluster
+Deploy a `LoadBalancer` type of service with the following manifest.
+```shell
+$ cat <<EOF >sample-svc.yaml
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: corgi-test
+  labels:
+    corgi: test
+spec:
+  selector:
+    matchLabels:
+      corgi: test
+  template:
+    metadata:
+      labels:
+        corgi: test
+    spec:
+      containers:
+        - name: nginx
+          image: harbor-repo.vmware.com/dockerhub-proxy-cache/library/nginx
+          ports:
+            - containerPort: 80
+
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: corgi-test
+spec:
+  type: LoadBalancer
+  selector:
+    corgi: test
+  ports:
+    - nodePort: 30008
+      port: 80
+      targetPort: 80
+EOF
+```
+After that, verify that the service has been assigned an external IP using `kubectl` and the AVI controller UI.
```

Comment:
Not get into details, but feels like "AVI controller UI" -> "AVI controller API"?

Reply:
The AVI controller provides the UI dashboard when you go to `http://<avi_controller_ip>`, gonna make it more clear in this sentence.